docker/micro-cluster-lab/scripts_db/mysql/init/test_create.sql

```diff
@@ -0,0 +1,2 @@
+Create schema nk;
+Create table test(id INT NOT NULL, name Varchar NOT NULL, value INT NOT NULL);
```

Comment:
Do you need this test script?
If do, then:
Rename schema to more common name.
Write SQL statements in upper case.

Reply:
I change this script